manifests/params.pp

```diff
@@ -38,17 +43,42 @@
       } else {
         $package_dependencies   = [ 'libruby', 'libcurl4-openssl-dev' ]
       }
+      $passenger_conf_template = 'passenger/passenger-enabled.erb'
+      $passenger_load_template = 'passenger/passenger-load.erb'
     }
     'redhat': {
-      $package_dependencies   = [ 'libcurl-devel', 'openssl-devel', 'zlib-devel' ]
-      $package_name           = 'passenger'
-      $passenger_package      = 'passenger'
-      $gem_path               = '/usr/lib/ruby/gems/1.8/gems'
-      $gem_binary_path        = '/usr/lib/ruby/gems/1.8/gems/bin'
-      $passenger_root         = "/usr/lib/ruby/gems/1.8/gems/passenger-${passenger_version}"
-      $mod_passenger_location = "/usr/lib/ruby/gems/1.8/gems/passenger-${passenger_version}/ext/apache2/mod_passenger.so"
+      $install_config          = true
+      $package_name            = 'passenger'
+      $passenger_package       = 'passenger'
+      $package_dependencies    = [ 'libcurl-devel', 'openssl-devel', 'zlib-devel' ]
+      $passenger_conf_template = 'passenger/passenger-conf.erb'
+      $passenger_load_template = undef
+      
+      # not sure what fedora operatingsystemmajrelease is so only pulling RHEL 7.x /Centos 7.x
+      if versioncmp($::operatingsystemmajrelease, '7') >= 0 and versioncmp($::operatingsystemmajrelease, '8') < 0 {
```

Comment:
This can just be `versioncmp($::operatingsystemmajrelease, '7') == 0`, no? Unless EL7 is one of the weird platforms that doesn't have `operatingsystemmajrelease` be _just_ the major release...